data/net/minecraft/client/tutorial/Tutorial.mapping

```diff
@@ -1,4 +1,9 @@
 CLASS net/minecraft/client/tutorial/Tutorial
+	METHOD <init> (Lnet/minecraft/client/Minecraft;)V
+		ARG 1 mc
+	METHOD addTimedToast (Lnet/minecraft/client/gui/components/toasts/TutorialToast;I)V
+		ARG 1 toast
+		ARG 2 duration
```

Comment:
Parameter names should follow the name of the field they're (eventually) assigned to.
```suggestion
		ARG 2 durationTicks
```